docs/operator-guides/reset.md

```diff
@@ -7,7 +7,15 @@ Airbyte allows you to reset all streams in the connection, some, or only a singl
 A sync will automatically start after a completed reset, which commonly backfills all historical data.
 
 ## Performing a Reset
-To perform a reset, select `Reset your data` in the UI on a connection's status or job history tabs. You will also be prompted to reset affected streams if you edit any stream settings to ensure data continues to sync accurately.
+To perform a full reset that resets all your streams, select `Reset your data` in the UI on a connection's status or job history tabs by selecting the three grey dots next to "Sync now". 
+
+To reset a single stream, navigate to a Connection's status page and click the three grey dots next to any stream. This will perform a reset of only that stream. 
```

Comment:
I'd add something to the first sentence like "and select reset stream from the menu". I feel this way it sounds like clicking the three dots immediately will reset a stream.